File: BaekjoonOnlineJudge/Bronze1/1924/main.cpp
#include <iostream>
using namespace std;

int main(){
    int days[] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    int month, date, cnt = 0;
    string dow[] = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};

    cin >> month >> date;

    for(int i = 0 ; i <month -1 ; i++){
        cnt += days[i];
    }
    cnt += date;
    cout << dow[cnt % 7] << endl;
}

File: BaekjoonOnlineJudge/Bronze1/1924/main.py
days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

dow = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']

month, date = map(int, input().split())

cnt = 0
for i in range(month - 1):
    cnt += days[i]
cnt += date

print(dow[cnt % 7])
